packages/run-protocol/test/swingsetTests/governance/test-governance.js

```diff
@@ -71,6 +71,7 @@ test.before(async t => {
 async function main(t, argv) {
   const { kernelBundles, config } = t.context.data;
   const controller = buildVatController(config, argv, { kernelBundles });
+  t.teardown(E(controller).shutdown);
```

Comment:
why the `E(controller)` ?

Reply:
That test setup used that form and doesn't `await` the result of `buildVatController`, so I kept it intact 🤷‍♂️

Reply:
ah, ok. I happened to ask MarkM about that pattern today, from some unrelated code, and he said it was a wart that `E()` allowed the proxy-trap -returned function to be observed that way (`E.get` would not, and the tildot syntax would probably treat `x = target~.method; x()` quite differently than `x = target~.method()`, so he discouraged me from considering using it.

If `controller` doesn't last more than a few lines, I'd be willing to rewrite the code to do `const controller = await ..`. But yeah, I'm also ok with matching the existing pattern and calling it a day.